apps/preview/app/env.ts

```diff
@@ -0,0 +1,6 @@
+import type { KVNamespace } from "@cloudflare/workers-types";
+
+export interface Env {
+  __STATIC_CONTENT: KVNamespace;
+  SITES: KVNamespace;
+}
```

Comment:
problem is we wanted to keep cloudflare specific logic completely outside of this repo

Reply:
It's just copy paste of whole workspace. It doesn't work yet.
Gonna integrate with pages in other PR.

Reply:
And reading from kv is not much of a logic. Though we gonna keep writing outside of it for sure.

Reply:
sure, but we need to get it to a state where webstudio designer doesn't depend on workers in any way

Reply:
cloudflare pages are workers. And anyway this is only an environment like node.js and this binding is an env variable. It can be developed locally too.